### ISSUE TITLE
[stable/spinnaker] Unable to create 'Projects' after fresh spinnaker install on AWS/EKS/S3

### ISSUE THREAD

**Issue by theothermike** [helm/charts#14526]
**Describe the bug**

Unable to create 'Projects' once Spinnaker is installed using Helm on AWS/EKS w/S3

also filed with spinnaker, but not sure who's responsible:  https://github.com/spinnaker/spinnaker/issues/4480

**Version of Helm and Kubernetes**:
Helm: v2.14.0
Kubernetes: EKS 1.11

**Which chart**:

tried 1.10.0 and 1.11.1

**What happened**:

Spinnaker successfully installs

And enables the Kubernetes provider and discovers all the Applications.   I then connect using

```
kubectl port-forward --namespace spinnaker svc/spin-deck 9000:9000
```

When I try to create a 'Project' (tried many different inputs), the Popup says 'Upsert Project (00:00) Operation succeeded!" and then tries a redirect which shows some HTML over the background saying '<404> Please check your URL - we can't find any data for the project test2' 

I dug through the logs of all spinnaker pods, and the only relevant log lines I found in 'spin-gate':

```
2019-06-04 17:16:16.429  INFO 1 --- [trix-projects-7] c.n.s.g.s.internal.Front50Service        : ---> HTTP GET http://spin-front50.spinnaker:8080/v2/projects/test2
2019-06-04 17:16:16.472  INFO 1 --- [trix-projects-7] c.n.s.g.s.internal.Front50Service        : <--- HTTP 404 http://spin-front50.spinnaker:8080/v2/projects/test2 (42ms)
2019-06-04 17:16:16.474  WARN 1 --- [0.0-8084-exec-9] .m.m.a.ExceptionHandlerExceptionResolver : Resolved [com.netflix.spinnaker.gate.retrofit.UpstreamBadRequest: 404 ]
```

and front50

```
2019-06-04 17:15:43.598  WARN 1 --- [0.0-8080-exec-5] .m.m.a.ExceptionHandlerExceptionResolver : Resolved [com.netflix.spinnaker.front50.exception.NotFoundException: Object not found (key: test2)]
```

**What you expected to happen**:

To be able to create a Project

**How to reproduce it** (as minimally and precisely as possible):

```
helm install --name spinnaker --namespace spinnaker -f values.yaml
```

with values.yaml (redacted)

```
kubeConfig:
  enabled: true
  secretName: my-kubeconfig
  secretKey: config
  contexts:
    # Names of contexts available in the uploaded kubeconfig
    - cluster-staging
    - cluster-operations
  # This is the context from the list above that you would like
  # to deploy Spinnaker itself to.
  deploymentContext: cluster-operations
s3:
  enabled: true
  accessKey: redacted
  secretKey: redacted
  bucket: my-bucket
  region: my-region
  endpoint: http://my-bucket.s3-my-region.amazonaws.com
minio:
  enabled: false
```

**Anything else we need to know**:

**Comment by theothermike** [comment]
attn to the chart maintainers (or I'm not sure how everyone see's these!)

@viglesiasce
@lwander
@dwardu89
@paulczar

**Comment by theothermike** [comment]
As per comments on spinnaker/spinnaker#4480 This was resolved by changing the halyard image tag from 1.16.0 to 1.20.2

This may have other side effects though.  So far it works for me, I will continue to experiment with it, and leave this issue open until you can validate upgrading the helm chart to use 1.20.2 will not have other undesirable side effects.
